crates/compiler/can/src/abilities.rs

```diff
@@ -28,7 +28,7 @@ pub struct MemberVariables {
 
 /// The member and its signature is defined locally, in the module the store is created for.
 /// We need to instantiate and introduce this during solving.
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Did clippy catch this?

Reply:
yes it complains now if `PartialEq` is derived, and `Eq` can be but isn't

Reply:
wow@